### PR TITLE
feature: add single subscriber to specified group

### PR DESCRIPTION
**Pull request by vitormorales**
The https://developers.mailerlite.com/reference#add-single-subscriber endpoint also create/update the subscriber data including attributes like type.

The https://developers.mailerlite.com/reference#add-many-subscribers doesn't.

### CODE REVIEW

**Review by skoudoro — COMMENTED** [review]
Hi @vitormorales,

Good catch! Thank you for your contribution and for looking into this!

Overall, it looks good.

I just wonder whether your function `add_single_subscriber` should be inside `add_subscribers` because I am afraid that the user will be confused by these two functions.

Also, it will be good to document it by adding a small example in the README.rst

I will get back to you tomorrow. Thanks again!

File: mailerlite/group.py
Comment (skoudoro) [inline mailerlite/group.py:239]: I just learned something, `issubset` seems really useful! 👍🏾

Comment (vitormorales) [reply mailerlite/group.py:239]: Yay! And it's faster than checking if both keys are in dict.keys() like 
`'email' in subscribers_data.keys()`

**Comment by vitormorales** [comment]
> Hi @vitormorales,
> 
> Good catch! Thank you for your contribution and for looking into this!
> 
> Overall, it looks good.
> 
> I just wonder whether your function `add_single_subscriber` should be inside `add_subscribers` because I am afraid that the user will be confused by these two functions.
> 
> Also, it will be good to document it by adding a small example in the README.rst
> 
> I will get back to you tomorrow. Thanks again!

My opinion is: explicit is better. Also there are different endpoints with different effects. But it's your call! We can for sure merge the add_single_subscriber on the add_subscribers. LMK about it.

And sure! I'll add some documentation for this tomorrow. 
Thank you very much for your attention @skoudoro!

**Comment by skoudoro** [comment]
> My opinion is: explicit is better. Also, there are different endpoints with different effects

Ok, I agree, let's keep it as it is.

Can you just add the documentation by updating the README? After that, it will be ready to be merged!

Thank you

**Comment by vitormorales** [comment]
> > My opinion is: explicit is better. Also, there are different endpoints with different effects
> 
> Ok, I agree, let's keep it as it is.
> 
> Can you just add the documentation by updating the README? After that, it will be ready to be merged!
> 
> Thank you

I added the docs about it. Please double check? English isn't my first language. Sorry if I did any mistakes. 
Again thank you! I'll contribute more later! I can solve the issue about the env variable that you added.

Cheers!

**Review by skoudoro — APPROVED** [review]
Looks good to me. 

The English look fine to me.  I'm also not a native speaker so if there is a problem, let's hope that a native speaker will contribute and correct it 😅.

Thank you @vitormorales. merging

**Pull request merged**
